thumbor/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2
+FROM python:3.8
```

Comment:
which python version should we use?

Reply:
I'd go with 3.9, but any of 3.6+ will work (our CI tests those).